backend/app/src/main/resources/db/migration/V4__update_column_display_names.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE table_column_info  SET  title = 'PCN Clinic Status' WHERE title = 'FCN PCI Flag';
```

Comment:
To make it more robust, we should check if the table column title exists before updating the value. This is something I implemented in the first version. Do you think it is worth it?

Reply:
It is working and I don't think it is worth it at this point because I am thinking of consolidate all the changes into one script, V2__seed_data_in_tables.sql, when the app is more stable.